cinema/views.py

```diff
@@ -15,10 +16,12 @@ def movies_list(request: HttpRequest) -> Response:
         return Response(serializer.data, status=status.HTTP_200_OK)
     else:
         serializer = MovieSerializer(data=request.data)
-        if serializer.is_valid():
+        try:
+            serializer.is_valid(raise_exception=True)

```

Comment:
you don't need `try`, `if` raise_exception means that it will send 400 response if serializer is not valid